src/core_plugins/kibana/public/management/sections/settings/components/field/field.js

```diff
@@ -462,30 +462,40 @@ export class Field extends PureComponent {
   }
 
   renderDescription(setting) {
```

Comment:
Sorry for the whitespace deltas in this file. The important change is here. The description used to always render html (dangerously, I might add), but now it will render a React element if one is provided. If not, then it falls back to the original behavior. I need this so that I can render EuiLinks within the usage data description